ci-operator/templates/openshift/installer/cluster-launch-installer-upi-e2e.yaml

```diff
@@ -353,6 +353,17 @@ objects:
         sshKey: |
           ${SSH_PUB_KEY}
         EOF
+            export OVA_URL="$(jq -r '.baseURI + .images["vmware"].path' /var/lib/openshift-install/rhcos.json)"
+            echo "Creating a template for the VMs from ${OVA_URL}..."
+            curl -O -L "${OVA_URL}"
```

Comment:
This should happen in some tmp dir:
```
Creating a template for the VMs from https://releases-art-rhcos.svc.ci.openshift.org/art/storage/releases/ootpa/410.8.20190412.1/rhcos-410.8.20190412.1-vmware.ova...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0Warning: Failed to create the file rhcos-410.8.20190412.1-vmware.ova: 
Warning: Permission denied
```

Reply:
Fixed in 9c7f8820...ae387060. OVA is now written out to `/etc/rhcos/rhcos.ova`.